charts/argo-cd/templates/dex/deployment.yaml

```diff
@@ -61,6 +61,8 @@ spec:
         imagePullPolicy: {{ default .Values.global.image.imagePullPolicy .Values.dex.image.imagePullPolicy }}
         command:
         - /shared/argocd-dex
+        - --logformat={{ default .Values.global.logging.format .Values.dex.logFormat }}
+        - --loglevel={{ default .Values.global.logging.level .Values.dex.logLevel }}
```

Comment:
✏️ 
There is no log config for dex on configMap, but I will check actual code later... 🤔 

https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cmd-params-cm.yaml

Reply:
It looks that [argocd-cmd-params-cm](https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cmd-params-cm.yaml) does not support dex log config, so it's ok to define on command, instead of [here](https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/_helpers.tpl#L197-L200). 🤔